definitions/bulk-action/json/CaseEvent.json

```diff
@@ -74,6 +74,7 @@
     "PreConditionState(s)": "Listed",
     "PostConditionState": "*",
     "SecurityClassification": "Public",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/generate-document?templateId=FL-DIV-GNO-ENG-00059.docx&documentType=caseListForPronouncement&filename=caseListForPronouncement",
```

Comment:
caseListForPronouncement documentType doesn't exist right?
Document type must be one of https://github.com/hmcts/div-ccd-definitions/blob/master/definitions/divorce/json/FixedLists.json#L220. If caseListForPronouncement is a valid type for the business, please add it to the FixedList enums.

Reply:
Good catch, @tchow8. I've brought the enum to Bulk List with only this document for now.